patternfly/patternfly-elements
# feat: Add support for empty local variables

Link(s) to demo pages where these udpates can be viewed (only listed components that have changes):

- [pfe-accordion](https://deploy-preview-854--happy-galileo-ea79c4.netlify.app/elements/pfe-accordion/demo/)
- [pfe-autocomplete](https://deploy-preview-854--happy-galileo-ea79c4.netlify.app/elements/pfe-autocomplete/demo/)
- [pfe-avatar](https://deploy-preview-854--happy-galileo-ea79c4.netlify.app/elements/pfe-avatar/demo/)
- [pfe-badge](https://deploy-preview-854--happy-galileo-ea79c4.netlify.app/elements/pfe-badge/demo/)
- [pfe-band](https://deploy-preview-854--happy-galileo-ea79c4.netlify.app/elements/pfe-band/demo/)
- [pfe-card](https://deploy-preview-854--happy-galileo-ea79c4.netlify.app/elements/pfe-card/demo/)  
- [pfe-cta](https://deploy-preview-854--happy-galileo-ea79c4.netlify.app/elements/pfe-cta/demo/)
- [pfe-dropdown](https://deploy-preview-854--happy-galileo-ea79c4.netlify.app/elements/pfe-dropdown/demo/)
- [pfe-health-index](https://deploy-preview-854--happy-galileo-ea79c4.netlify.app/elements/pfe-health-index/demo/)
- [pfe-icon](https://deploy-preview-854--happy-galileo-ea79c4.netlify.app/elements/pfe-icon/demo/)
- [pfe-jump-links](https://deploy-preview-854--happy-galileo-ea79c4.netlify.app/elements/pfe-jump-links/demo/)
- [pfe-markdown](https://deploy-preview-854--happy-galileo-ea79c4.netlify.app/elements/pfe-markdown/demo/)
- [pfe-modal](https://deploy-preview-854--happy-galileo-ea79c4.netlify.app/elements/pfe-modal/demo/)
- [pfe-navigation](https://deploy-preview-854--happy-galileo-ea79c4.netlify.app/elements/pfe-navigation/demo/)
- [pfe-page-status](https://deploy-preview-854--happy-galileo-ea79c4.netlify.app/elements/pfe-page-status/demo/)
- [pfe-progress-indicator](https://deploy-preview-854--happy-galileo-ea79c4.netlify.app/elements/pfe-progress-indicator/demo/)
- [pfe-select](https://deploy-preview-854--happy-galileo-ea79c4.netlify.app/elements/pfe-select/demo/)
- [pfe-styles](https://deploy-preview-854--happy-galileo-ea79c4.netlify.app/elements/pfe-styles/demo/)
- [pfe-tabs](https://deploy-preview-854--happy-galileo-ea79c4.netlify.app/elements/pfe-tabs/demo/)
- [pfe-toast](https://deploy-preview-854--happy-galileo-ea79c4.netlify.app/elements/pfe-toast/demo/)

### For component fixes and features

* [Link to issue detailing the request](https://github.com/patternfly/patternfly-elements/issues/903)

### What has changed and why

* Refactor of PFE functions responsible for outputting correct variables and fallbacks such that local variables are empty by default

### Testing instructions

Be sure to include detailed instructions on how your update can be tested by another developer.

1. Ensure that components visually look the same
2. Spot check various existing local variables to make sure they still work
3. Check theme variables to ensure they still work. 

#### Browser requirements

Your component should work in all of the following environments:


### Ready-for-merge Checklist

Check off items as they are completed.  Feel free to delete items if they are not applicable to your PR.




https://rally1.rallydev.com/#/270861059696d/detail/userstory/390981143804
